wiki/osu!tourney/osu!lazer_tournament_client/en.md

```diff
@@ -0,0 +1,66 @@
+# osu!lazer tournament client
+
+**osu!lazer tournament client** is the official client that overlays osu!tourney.
+
+Users encountering problems with the client may [create an issue](https://github.com/ppy/osu/issues) or send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
+
+## Setup
+
+To start the osu!lazer tournament client, you need to specify a launch argument to the executable.
+
+To do this, create a new shortcut on the desktop and set the location for it to `%LOCALAPPDATA%\osulazer\osu!.exe --tournament`. This will make that specific shortcut start up osu!lazer in the tournament client mode.
```

Comment:
seems like an odd way to introduce making the shortcut, it's written as if this is the only way to "specify a launch argument to the executable"